src/server/dragonfly_test.cc

```diff
@@ -91,6 +92,28 @@ TEST_F(DflyEngineTest, Sds) {
   sdsfreesplitres(argv, argc);
 }
 
+class DflyRenameCommandTest : public DflyEngineTest {
+ protected:
+  DflyRenameCommandTest() : DflyEngineTest() {
+    // remame flushall to myflushall, flushdb command will not be able to execute
+    absl::SetFlag(&FLAGS_rename_command, "flushall=myflushall,flushdb=,");
+  }
+};
+
+TEST_F(DflyRenameCommandTest, RenameCommand) {
+  Run({"set", "a", "1"});
+  ASSERT_EQ(1, CheckedInt({"dbsize"}));
+  // flushall should not execute anything and should return error, as it was renamed.
+  RespExpr resp = Run({"flushall"});
+  ASSERT_THAT(resp, ErrArg("unknown command `FLUSHALL`"));
+  ASSERT_EQ(1, CheckedInt({"dbsize"}));
+  resp = Run({"myflushall"});
+  ASSERT_EQ(resp, "OK");
+  ASSERT_EQ(0, CheckedInt({"dbsize"}));
+  resp = Run({"flushdb", "0"});
+  ASSERT_THAT(resp, ErrArg("unknown command `FLUSHDB`"));
```

Comment:
What will happen if you tried to execute the empty command (i.e. `flushdb`)? What prevents it from running? Don't we need special handling for that in `CommandRegistry::operator<<`?